src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteForSchemeRequestHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace EA.Weee.RequestHandlers.AatfEvidence
+{
+    using System.Threading.Tasks;
+    using Domain.Evidence;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.AatfEvidence;
+    using EA.Weee.DataAccess.DataAccess;
+    using Prsd.Core;
+    using Prsd.Core.Mediator;
+    using Requests.AatfEvidence;
+    using Security;
+
+    public class GetEvidenceNoteForSchemeRequestHandler : IRequestHandler<GetEvidenceNoteRequest, EvidenceNoteData>
```

Comment:
Create new GetEvidenceNoteBySchemeRequest and this should be IRequestHandler<GetEvidenceNoteBySchemeRequest, EvidenceNoteData>

Reply:
request created and used by mediator

Reply:
modification done